03-Mains/Temperature_Salinity_Diagram.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "65fe6a4e-f751-40c3-80ed-12c66e728784",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        # especially in the vertical dimension
</pre><p>Can grid cell thicknesses vary in a dimension other than the vertical?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/619/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>